tests/test_permissions.py

```diff
@@ -64,3 +72,29 @@ def test_permission_not_equal(self):
         """
         self.assertTrue(permissions.RAT != permissions.TECHRAT)
         self.assertTrue(permissions.DISPATCH != permissions.ORANGE)
+
+    @async_test
+    async def test_restricted_command(self):
+        await Commands.trigger("!restricted", "some_recruit", "#somechannel")
+        self.assertIn({
+            "target": "#somechannel",
+            "message": permissions.OVERSEER.denied_message
+        }, self.bot.sent_messages)
+
+        await Commands.trigger("!restricted", "some_ov", "#somechannel")
```

Comment:
It appears you are testing the same bit of code with different parameters, these should be marked as sub-tests